packages/plugins/plugin-map/src/components/MapContainer.tsx

```diff
@@ -30,6 +31,12 @@ export const MapContainer = ({ role, type: _type = 'map', map, ...props }: MapCo
   const schema = useSchema(space, map?.view?.target?.query.type);
   const rowsForType = useQuery(space, schema ? Filter.schema(schema) : undefined);
 
+  // TODO(burdon): Do something with selected items (ids). (Correlate against `rowsForType`).
+  const selectedItems = useSelectedItems(map!.view!.target!.query.type!);
+  useEffect(() => {
+    console.log('Selected items changed:', selectedItems);
+  }, [selectedItems]);
```

Comment:
@richburdon, the selection for type is now available. This will show you the selection from any Kanban / Table that shares the typename.a